pkg/models/terminal/terminal.go

```diff
@@ -224,3 +355,49 @@ func (t *terminaler) HandleSession(shell, namespace, podName, containerName stri
 
 	session.Close(1, "Process exited")
 }
+
+func (t *terminaler) HandleShellAccessToNode(nodename string, conn *websocket.Conn) {
+	ch := make(chan bool, 1)
+	once.Do(func() {
```

Comment:
once.Do had better add to func init section.

Reply:
If there is no shell access request, this variable will not be used, so I put it here

Reply:
> If there is no shell access request, this variable will not be used, so I put it here

I think one variable doesn't take up much resources.

Reply:
OK.I'll move it to init section